### PR TITLE
updating twilio voice example

### PR DESCRIPTION
**Pull request by benlower**
Removed librosa and replaced with soxr. tweaks to logging. explicitly added fixie dependency. changed to run on "0.0.0.0" so it will work in a container (was  "localhost" prior).

### CODE REVIEW

File: examples/twilio_stream_example.py [hunk 1]
```diff
@@ -28,9 +32,7 @@ async def start(self, sample_rate: int, num_channels: int):
 
     async def write(self, chunk: bytes) -> None:
         sample = numpy.frombuffer(chunk, numpy.int16).astype(numpy.float32)
-        resampled = librosa.resample(
-            sample, orig_sr=self._sample_rate, target_sr=8000
-        ).astype(numpy.int16)
+        resampled = soxr.resample(sample, 48000, 8000).astype(numpy.int16)
```
Comment (juberti) [inline examples/twilio_stream_example.py:35]: use `self._sample_rate` as the input rather than `48000`

File: examples/twilio_stream_example.py [hunk 2]
```diff
@@ -28,9 +32,7 @@ async def start(self, sample_rate: int, num_channels: int):
 
     async def write(self, chunk: bytes) -> None:
         sample = numpy.frombuffer(chunk, numpy.int16).astype(numpy.float32)
```
Comment (juberti) [inline examples/twilio_stream_example.py:34]: you can remove the .astype here as soxr can handle int16 data

File: examples/twilio_stream_example.py [hunk 3]
```diff
@@ -88,25 +94,27 @@ async def on_sink_data(data):
     @session.on("state")
     async def on_state(state):
         if state == types.SessionState.LISTENING:
```
Comment (juberti) [inline examples/twilio_stream_example.py:96]: probably a good idea to just emit each state, e.g., logging.info(f"State: {state}")

File: examples/twilio_stream_example.py [hunk 4]
```diff
@@ -88,25 +94,27 @@ async def on_sink_data(data):
...
+            logging.info("State:  Listening")
         elif state == types.SessionState.THINKING:
-            print("Agent:  ", end="\r")
+            logging.info("State:  Thinking")
 
     @session.on("input")
     async def on_input(text, final):
-        print("User:  " + text, end="\n" if final else "\r")
+        if final:
+            logging.info("User:  " + text)
```
Comment (juberti) [inline examples/twilio_stream_example.py:104]: remove extra space after colon

File: examples/twilio_stream_example.py (hunk 1 above)
Comment (benlower) [reply examples/twilio_stream_example.py:35]: done

File: examples/twilio_stream_example.py (hunk 2 above)
Comment (benlower) [reply examples/twilio_stream_example.py:34]: done

File: examples/twilio_stream_example.py (hunk 3 above)
Comment (benlower) [reply examples/twilio_stream_example.py:96]: done

File: examples/twilio_stream_example.py (hunk 4 above)
Comment (benlower) [reply examples/twilio_stream_example.py:104]: done

**Review by juberti — APPROVED** [review]

**Pull request merged**